src/test/common/variables/envVarsService.unit.test.ts

```diff
@@ -206,7 +205,7 @@ PYTHON=${BINDIR}/python3\n\
                 variablesService.mergeVariables(vars1, vars2);
 
                 expect(Object.keys(vars1)).lengthOf(2, 'Source variables modified');
-                expect(Object.keys(vars2)).lengthOf(5, 'Variables not merged');
+                expect(Object.keys(vars2)).lengthOf(3, 'Variables not merged');
```

Comment:
PATH & PYTHONPATH should not be merged (we validate this by checking the expected 3 properties)